index.js

```diff
@@ -20,7 +20,70 @@ mongoose
     return self.connection.dropDatabase();
   })
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+    //Iteration 1
+    // Recipe.create(data[0])
+    //   .then((res) => {
+    //     console.log("Data added", res);
+    //   })
+    //   .catch((err) => {
+    //     console.log("Something went wrong", err);
+    //   });
+
+
+    let insertRecipes = Recipe.insertMany(data);
+    // let updateRecipe;
+    // let deleteRecipe;
+
+    insertRecipes
+      .then((res) => {
+        console.log("Data added");
```

Comment:
part of the iteration was to show the title of the elements inserted, using a loop on the res